### PR TITLE
[DX-443] add custom 404 page template 

### PR DESCRIPTION
**Pull request by yurisasuke**
[DX-443]

[DX-443]: https://tyktech.atlassian.net/browse/DX-443?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

https://deploy-preview-3737--tyk-docs.netlify.app/docs/nightly/test-404

### CODE REVIEW

**Comment by letzya** [comment]
@Keithwachira are you sure we can write in _redirect? Netlify overwrites it while building.

**Comment by yurisasuke** [comment]
> @Keithwachira are you sure we can write in _redirect? Netlify overwrites it while building.

@letzya it is still work in progress. I was following this tutorial from Hugo website https://gohugo.io/templates/404/

https://gohugo.io/templates/404/#automatic-loading

**Comment by letzya** [comment]
FYI: last time it also didn't work #2080

File: tyk-docs/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[[redirects]]
+  from = "/*"
+  to = "/404.html"
```
Comment (letzya) [inline tyk-docs/netlify.toml]:
```suggestion
  to = "/404"
```

**Comment by yurisasuke** [comment]
replaced not needed

**Pull request closed without merging**